frb_rust/src/third_party/wasm_bindgen/worker_pool.rs

```diff
@@ -172,9 +216,32 @@ impl WorkerPool {
     /// Returns any error that may happen while a JS web worker is created and a
     /// message is sent to it.
     // NOTE: It is originally named `execute`, but rename to align with crate `threadpool`
-    fn execute_raw(&self, closure: TransferClosure<JsValue>) -> Result<Worker, JsValue> {
-        let worker = self.worker()?;
-        closure.apply(&worker).map(|_| worker)
+    fn execute_raw(&self, closure: TransferClosure<JsValue>) -> Result<ExecuteResult, JsValue> {
```

Comment:
TODO(@fzyzcjy): carefully recheck the core logic again after code cleanup

(you do not need to do anything to this comment, it is just a reminder for myself)